lmms_eval/tasks/docqa_hu_syn/docqa_hu_syn.yaml

```diff
@@ -0,0 +1,23 @@
+dataset_path: nhiremath/HungarianDocQA_IT_SynQA
+task: "docqa_hu_syn"
+output_type: generate_until
+test_split: test
+doc_to_visual: !function utils.docvqa_doc_to_visual
+doc_to_text: !function utils.docvqa_doc_to_text
+doc_to_target: !function utils.docvqa_doc_to_target
+generation_kwargs:
+  max_new_tokens: 128
+  temperature: 0
+  do_sample: False
+model_specific_prompt_kwargs:
+  default:
+    pre_prompt: "\nVálaszold meg a kérdést. "
+    post_prompt: "Válasz magyarul:"
```

Comment:
Nidhi's prompts already have "answer in hungarian" at the end. Can we account for that? https://huggingface.co/datasets/nhiremath/HungarianDocQA_IT_SynQA